packages/react/src/components/ProductDataList/src/components/ProductDataListItem/ProductDataListItem.tsx

```diff
@@ -14,6 +21,9 @@ const ProductDataListItem: React.FC<ProductDataListItemProps> = ({
   withDivider = false,
   onRemove,
   children,
+  tagIcon,
```

Comment:
```suggestion
  tagIcon: TagIcon,
```

Y despues en el codigo
<TagIcon />